### ISSUE TITLE
Breaking change, no documentation for it: Invalid type for parameter FeedbackForwardingEmailAddress, value: None, type: <class 'NoneType'>, valid types: <class 'str'>

### ISSUE THREAD

**Issue by metalaureate** [django-ses/django-ses#278]
Any tips on how to fix  in 3.4.0

Invalid type for parameter FeedbackForwardingEmailAddress, value: None, type: <class 'NoneType'>, valid types: <class 'str'> ? Sorry if it is obvious somewhere and I missed it.

```
 email_processor = EmailMultiAlternatives(subject, text_content, from_email, [email])
            email_processor.attach_alternative(html_content, "text/html")
            try:
                email_processor.send()
                NotificationLog.objects.create(email=email, send_at=timezone.now())
            except Exception as e:
                logger.error(e)
```

Thank you!

**Comment by pcraciunoiu** [comment]
@peidrao @rodrigondec have you come across this? Is it related to the change in #276 ?

Does this issue happen in v3.3.0 too @metalaureate ? If not, it could have to do with the from address not being verified. See here: https://github.com/django-ses/django-ses/pull/276#discussion_r1169200001

**Comment by metalaureate** [comment]
No does not happen in 3.3.0

**Comment by pcraciunoiu** [comment]
It sounds like a value needs to be set for `AWS_SES_RETURN_PATH` even though it's expected to be optional. That might be your short-term fix for now.

@peidrao can you fix so it can be optional in a follow-up?

**Comment by rodrigondec** [comment]
We forgot to check if `email_feedback is None` to use the arg `FeedbackForwardingEmailAddress`

https://github.com/django-ses/django-ses/blob/82956612731a8cbda8f017d136766f722a581bd7/django_ses/__init__.py#L247-L270

I think this would resolve the issue

```python
     def _get_v2_parameters(self, message, source, email_feedback): 
         """V2-Style raw payload for `send_email`. 
  
         https://boto3.amazonaws.com/v1/documentation/api/1.26.31/reference/services/sesv2.html#SESV2.Client.send_email 
         """ 
         params = dict(  
             FromEmailAddress=source or message.from_email, 
             Destination={ 
                 'ToAddresses': message.recipients() 
             }, 
             Content={ 
                 'Raw': { 
                     'Data': dkim_sign(message.message().as_string(), 
                                       dkim_key=self.dkim_key, 
                                       dkim_domain=self.dkim_domain, 
                                       dkim_selector=self.dkim_selector, 
                                       dkim_headers=self.dkim_headers) 
                 } 
             } 
         ) 
         if self.ses_from_arn or self.ses_source_arn: 
             params['FromEmailAddressIdentityArn'] = self.ses_from_arn or self.ses_source_arn 
         if email_feedback is not None:
             params['FeedbackForwardingEmailAddress'] = email_feedback
         return params 
```

**Comment by pcraciunoiu** [comment]
@rodrigondec skipping the param when it's None makes sense to me. Go ahead and PR that and I'll release a patch version. If you can't reproduce the issue by setting to `None` on v3.4.0, we can have the reporting user test it.

But worth a try if you can reproduce it with `None` (or omitting the setting).  Make sure `USE_SES_V2` is set

**Comment by rodrigondec** [comment]
Problem reproduced

```
Traceback (most recent call last):
File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/django/core/mail/message.py", line 284, in send
    return self.get_connection(fail_silently).send_messages([self])
  File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/django_ses/__init__.py", line 166, in send_messages
    response = (self.connection.send_email(**kwargs)
  File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/botocore/client.py", line 530, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/botocore/client.py", line 919, in _make_api_call
    request_dict = self._convert_to_request_dict(
  File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/botocore/client.py", line 990, in _convert_to_request_dict
    request_dict = self._serializer.serialize_to_request(
  File "/home/rodrigondec/prog/imobanco/income-back/.venv/lib/python3.9/site-packages/botocore/validate.py", line 381, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter FeedbackForwardingEmailAddress, value: None, type: <class 'NoneType'>, valid types: <class 'str'>
```

We verified the solution performing a regression test using the install from the branch on our fork https://github.com/imobanco/django-ses/tree/fix/feedback_none.

**Comment by pcraciunoiu** [comment]
This has been fixed in #279 and released as v3.4.1 on PyPi

Thanks for the quick fix!

**Comment by metalaureate** [comment]
Thank you!
